llvm/lib/Target/RISCV/RISCVProcessors.td

```diff
@@ -490,6 +490,7 @@ def TENSTORRENT_ASCALON_D8 : RISCVProcessorModel<"tt-ascalon-d8",
                                                   FeatureUnalignedScalarMem,
                                                   FeatureUnalignedVectorMem]),
                                                  [TuneNoDefaultUnroll,
+                                                  TuneFastVRGather,
```

Comment:
Looks like an extraneous new line

Reply:
Joining with next line would definitely push that past 80 columns, is that OK?

Reply:
Ah ignore my comment, my browser window had truncated the rest of the line, sorry!